src/Microsoft.AspNetCore.Server.IISIntegration/IISMiddleware.cs

```diff
@@ -76,6 +76,13 @@ public IISMiddleware(RequestDelegate next, ILoggerFactory loggerFactory, IOption
                 return;
             }
 
+            var bodySizeFeature = httpContext.Features.Get<IHttpMaxRequestBodySizeFeature>();
+            if (bodySizeFeature != null)
+            {
+                // IIS already limits this, no need to do it twice.
+                bodySizeFeature.MaxRequestBodySize = null;
```

Comment:
Should we log a warning? It might help people figure out why the limit they configured isn't having any effect.